.github/workflows/linux_wheel.yml

```diff
@@ -0,0 +1,69 @@
+name: Build Linux Wheels
+
+on:
+  pull_request:
+
+jobs:
+  generate-build-matrix:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    container:
+      image: python:3.9
+    steps:
+      - name: Clone pytorch/pytorch
+        uses: actions/checkout@v2
+      - id: set-matrix
+        run: |
+          # outputting for debugging purposes
+          python .github/scripts/generate_binary_build_matrix.py
+          MATRIX=$(python .github/scripts/generate_binary_build_matrix.py)
+          echo "::set-output name=matrix::${MATRIX}"
```

Comment:
Is there a limit to the size of a string that can be passed this way as an output to the next job?

Reply:
No limit afaik